Add Gradient Styles for Text View

vinothvino42 opened the pull request:
Ticket: #583 

In this PR, I added gradient styles for the text view.

```yaml
Text:
  styles:
    textStyle:
      fontSize: 20,
      color: green,
      fontWeight: w900,
      gradient:
        {
          colors: [0xFF675FF3, 0xFF4B93E7, 0xFF48CFD7],
          start: centerLeft,
          end: centerRight,
        }
  text: Ensemble App Developer
```
NOTE: Text `color` will be excluded from the text styles if you have a `gradient`.

<img src="https://github.com/EnsembleUI/ensemble/assets/12869588/e4965c48-f960-418c-ad9b-80e79489259b" width="350" />

vusters reviewed (commented):
Please remember to add the schema. Make sure to reuse the same schema style that Row/Column's gradientBackground use.

vinothvino42 commented:
> Please remember to add the schema. Make sure to reuse the same schema style that Row/Column's gradientBackground use.

Can you just review my PR? I updated new changes and added schema
